x-pack/plugins/monitoring/public/services/breadcrumbs_provider.js

```diff
@@ -17,90 +18,116 @@ const createCrumb = (url, label, testSubj) => {
 
 // generate Elasticsearch breadcrumbs
 function getElasticsearchBreadcrumbs(mainInstance) {
+  const esLabel = i18n.translate('xpack.monitoring.breadcrumbs.esLabel', { defaultMessage: 'Elasticsearch' });
```

Comment:
I'm still not clear on this. Are we translating `Elasticsearch` or not?

Reply:
Thanks. No, we don't translate this label.